src/main/java/org/synyx/urlaubsverwaltung/application/service/ApplicationReminderMailService.java

```diff
@@ -61,6 +66,18 @@ void sendWaitingApplicationsReminderNotification() {
         }
     }
 
+    void sendUpcomingApplicationsReminderNotification() {
+
+        final ApplicationSettings applicationSettings = settingsService.getSettings().getApplicationSettings();
+        if (applicationSettings.isRemindForUpcomingApplications()) {
+            final LocalDate dateForUpcomingApplications = LocalDate.now(clock).plusDays(applicationSettings.getDaysBeforeRemindForUpcomingApplications());
+            final List<ApplicationStatus> allowedStatuses = List.of(ALLOWED, ALLOWED_CANCELLATION_REQUESTED, TEMPORARY_ALLOWED);
```

Comment:
hier wird es sich imho anbieten separate E-Mails zu senden.

Für 
* `ALLOWED` Urlaub gibt's diese Info
* `ALLOWED_CANCELLATION_REQUESTED` brauche ich selbst vielleicht gar keine Info? 🤔
* `TEMPORARY_ALLOWED` eine Info, dass ich nochmal nachfragen muss vielleicht?

fürs _jetzt_ finde ich wie es hier gelöst wurde auf jeden fall gut. Besser als gar keine Info :-)

Reply:
Guter Hinweis. Überlege ich nochmal

Reply:
Bei `ALLOWED_CANCELLATION_REQUESTED` und `TEMPORARY_ALLOWED` warte ich als Antragssteller ja drauf das jemand anderes etwas tut. Da ist echt die Frage was man in so einem Fall macht. Vielleicht direkt dem Freigabeberechtigen eine Mail senden?

Reply:
bei `ALLOWED_CANCELLATION_REQUESTED` ist der Urlaub an sich genehmigt und es wird lediglich die Anfrage gestellt ihn zu stornieren. Wenn hier nicht agiert wird heisst es ja das der Urlaub stattfindet. Hier würde ich, so wie bei den wartenden Anträgen, auch ein separater Reminder verschicken bzw den für wartende Aufbohren.

Bei `TEMPORARY_ALLOWED` bin ich mir gerade unsicher ob der dann schon genhemigt ist in diesem Zustand oder erst wenn er auf `ALLOWED` steht. Hat wer Doku für mich?`Ich finde nichts @honnel

Reply:
* `ALLOWED_CANCELLATION_REQUESTED`: Finde ich gut was du schreibst :+1: 
* `TEMPORARY_ALLOWED`: Mehr als [diese Doku](https://github.com/synyx/urlaubsverwaltung/wiki/Fragen-zu-Urlaub#technische-dokumentation-f%C3%BCr-application-status) gibt es leider nicht. Ich denke, es ist Auslegungssache. Allerdings macht es ja wenig Sinn einen zweistufigen Prozess zu fahren, bei dem die zweite Stufe dann ignoriert wird. Aber ggf. gibt es Konstellationen wie, als Chef kann ich noch ein Veto einlegen, aber wenn ich nichts Sage ist es schon okay so. Würde ihn daher auch eher als genehmigt verstehen als nicht genehmigt.

Reply:
Dann lass uns das so mergen und noch Tickets für weitere Benachrichtigungen aufmachen.